libs/mark_predictions_on_image.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+from draw_utils import place_markers_on_images
+from utils import list_dirs
+
+test_data_dir = os.path.dirname(os.path.realpath(__file__)) + "/../test"
+ml_dir = "{}/tmp/pipeline2/machine_learning".format(test_data_dir)
+
+
+def get_ml_predictions_dirs(directory):
+    ml_dir_sets = list_dirs(directory)
+    return ["{}/{}".format(ml_dir, x) for x in ml_dir_sets]
+
+
+def prepare_data_from_predictions_csv(csv_dir_path):
```

Comment:
this should prepare data for draw_utils in a format that doesn't leak abstraction of machine learning models, 
e.g.
markers: [{
"x": 0.51,
"y": 0.34,
"color": "green"
"number": None
},
{
"x": 0.41,
"y": 0.44,
"color": "red"
"number": 1
}, ...]